internal/pkg/template/templates/environment/cf.yml

```
@@ -56,8 +56,93 @@ Conditions:
   ManagedAliases: !And
     - !Condition DelegateDNS
     - !Not [!Equals [ !Ref Aliases, "" ]]
+{{- if and .PublicHTTPConfig.ELBAccessLogs (not .PublicHTTPConfig.ELBAccessLogs.BucketName) }} #.PublicHTTPConfig.ELBAccessLogs.CreateBucket
+Mappings:
+  RegionalConfigs:
+    us-east-1:
+      ELBAccountId: '127311923021'
+      ArnPrefix: 'arn:aws:'
+    us-east-2:
+      ELBAccountId: '033677994240'
```

nit: in CFN they use a different casing
```suggestion
      ElbAccountId: '033677994240'
```